### DISCUSSION TITLE
Ruby'de miras alma işlemleri

### DISCUSSION THREAD

**Discussion by BunyaminAkdeniz** [19/classroom#32] [Ruby] [answered]
Merhaba hocam  @roktas. 
```ruby
class MemeliHayvanlar
  def initialize cins
    @cins = cins
  end
end

class EvcilHayvanlar
  def initialize pet_ismi
    @pet_ismi = pet_ismi
  end
end

class Kedi < Memeli < Evcil # => şöyle bir şeyi Ruby'de nasıl yapabilirim.
end

```

**Comment by roktas** [comment, marked as answer]
@BunyaminAkdeniz Ruby'de statik yolla "multiple inheritance" yapılamaz, bunun yerine "mixin" kullanılabilirsin.  Daha fazla yardımcı olurum ama biraz ileri bir konu bu.  NYP dersinde ayrıntısına gireceğim.  Bana hatırlat, o zaman anlatayım.

**Reply by BunyaminAkdeniz** [reply]
Tamamdır Hocam. Teşekkür ederim
